### PR TITLE
Minor refactorings.

### PR DESCRIPTION
**Pull request by sduskis**
These refactorings ought to ease the review of #521.

### CODE REVIEW

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java [hunk 1]
```diff
@@ -128,8 +128,8 @@ public Empty mutateRow(MutateRowRequest request) throws ServiceException {
 
   @Override
   public ListenableFuture<Empty> mutateRowAsync(MutateRowRequest request) {
-    return clientCallService.listenableAsyncCall(request,
-      channel.newCall(BigtableServiceGrpc.METHOD_MUTATE_ROW, CallOptions.DEFAULT));
+    return clientCallService.listenableAsyncCall(channel.newCall(BigtableServiceGrpc.METHOD_MUTATE_ROW, CallOptions.DEFAULT),
```
Comment (kevinsi4508) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java]: line seems too long, can you break it?

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 2]
```diff
@@ -519,15 +519,8 @@ private Channel wrapChannel(Channel channel) {
...
-          return false;
-        }
-        for (Mutation mut : mutateRowRequest.getMutationsList()) {
-          if (mut.getSetCell().getTimestampMicros() == -1) {
-            return false;
-          }
-        }
-        return true;
+        return mutateRowRequest != null
+            && !hasMissingTimeStamp(mutateRowRequest.getMutationsList());
```
Comment (kevinsi4508) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: allCellsHaveTimestamp?

Comment (sduskis) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: done

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java (hunk 1 above)
Comment (sduskis) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java]: done. I blame the eclipse refactoring too for this one.

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RetryingCall.java [hunk 3]
```diff
@@ -144,6 +146,9 @@ boolean retryCallAfterBackoff(
       final Metadata requestHeaders,
       final Listener<ResponseT> listener) {
     long sleepTimeout = BackOff.STOP;
+    if (backOff == null) {
```
Comment (kevinsi4508) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RetryingCall.java]: do we need to make it threadsafe?

Comment (sduskis) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RetryingCall.java]: It wasn't before, but that's probably a good idea.

**Comment by kevinsi4508** [comment]
LGTM

**Pull request merged**